src/interaction.js

```diff
@@ -16,21 +16,31 @@ limitations under the License.
 
 import {UserInteractionPlugin} from '@opentelemetry/plugin-user-interaction';
 
-const allowedEventTypes = {
-  click: true,
-  dblclick: true,
-  submit: true,
-  reset: true,
-  dragend: true,
-  drop: true,
-  ended: true,
-  pause: true,
-  play: true,
-  change: true,
-  mousedown: true,
-  mouseup: true,
-};
+const allowedEventTypes = [
+  // pointer
+  'click',
+  'dblclick',
+  'mousedown',
+  'mouseup',
 
+  // form
+  'submit',
+  'reset',
+  'change',
+
+  // drap & drop
+  'dragend',
+  'drop',
+
+  // media
+  'ended',
+  'pause',
+  'play',
+
+  // keyboard
```

Comment:
Two things: 1) the style change looks OK, but won't array.includes change our runtime here from O(1) to O(n) (n=number of allowed events), and (bigger) I specifically don't have key events in here because I found that real-world apps (including some of our beta customers) trigger a storm of these on things like search fields (each key triggers a lookup in the search cache/pre-fetched suggestions, etc. -> each key is a span).  I like Marten's idea of exposing this through config but I think the default needs to keep keydown/keyup out.

Reply:
You're right, I'm dropping key events from defaults. With regards to array vs. set, iterating over an array of a dozen, maybe two, of items will add measurable overhead to an event handler. That said, object with boolean values makes more sense for configurability.